### PR TITLE
fix(ui-buttons): remove underline from disabled Button with href

### PR DESCRIPTION
**Pull request by ToMESSKa**
INSTUI-4540

### CODE REVIEW

File: packages/ui-buttons/src/BaseButton/styles.ts [hunk 1]
```diff
@@ -401,26 +401,30 @@ const generateStyle = (
...
+          },
+          '&:focus': {
+            textDecoration: 'none'
+          },
+          '&:active > [class$=-baseButton__content]':
+            colorVariants[color!].active,
+          '&:hover > [class$=-baseButton__content]': colorVariants[color!].hover
+        }
+      : {
+          textDecoration: 'none'
```
Comment (matyasf) [inline packages/ui-buttons/src/BaseButton/styles.ts]: Please add here these:
```
          label: 'baseButton',
          appearance: 'none',
```

Label is needed, so it's easier to identity what this is in the DOM (also unit tests might use it), appearance: none might prevent some custom CSS to override things here (although I could not override it)

**Review by matyasf — CHANGES_REQUESTED** [review]
see my comment

File: packages/ui-buttons/src/BaseButton/styles.ts (hunk 1 above)
Comment (ToMESSKa) [reply packages/ui-buttons/src/BaseButton/styles.ts]: @matyasf I added them

**Review by matyasf — APPROVED** [review]

**Review by balzss — APPROVED** [review]
